contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -2127,11 +2127,13 @@ exec_stmt_exec_sp(PLtsql_execstate *estate, PLtsql_stmt_exec_sp *stmt)
 				int			save_nestlevel;
 				int			scope_level;
 				InlineCodeBlockArgs *args = NULL;
-
+				
```

Comment:
nit: spacing, I think that elsewhere in the code we don't have padding for empty lines.

Reply:
ack